src/session_concurrency/request_handler.cc

```diff
@@ -98,7 +148,7 @@ RequestHandler::SendRequest(
 
   const auto inputs{PrepareInputs(payload)};
 
-  RecordRequest(inputs, request_record);
+  request_record.start_time_ = std::chrono::system_clock::now();
```

Comment:
As mentioned [here](https://github.com/triton-inference-server/perf_analyzer/pull/302/files#r1972733350), request inputs are recorded slightly earlier, but we still record the start time as close to the client backend call as we can, here.

Reply:
qq: where is `SendRequest` being called? I only see `SendRequestAndWaitForResponse` being called.

Reply:
SendRequest is called within SendRequestAndWaitForResponse:

https://github.com/triton-inference-server/perf_analyzer/pull/302/files#diff-f51c753ff7d70252c18dafad999cf7ee58ac73fdedf113e66005f913d276c69fR83-R84

Reply:
Woops. Totally missed that haha 👍